ui/app/AppLayouts/Chat/popups/community/RemoteSelfDestructPopup.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import QtQml.Models 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Core.Theme 0.1
+
+import AppLayouts.Chat.panels.communities 1.0
+
+import utils 1.0
+
+StatusDialog {
+    id: root
+
+    property alias model: tokenHoldersPanel.model
+
+    property string collectibleName
+
+    signal selfDestructClicked(int tokensNumber)
+
+    QtObject {
+        id: d
+
+        readonly property int maxHeight: 560 // by design
+        property int tokensNumber: 0
+
+        function getVerticalPadding() {
+            return root.topPadding + root.bottomPadding
+        }
+
+        function getHorizontalPadding() {
+            return root.leftPadding + root.rightPadding
+        }
+
+        function calculateTotalTokensToDestruct() {
+            tokensNumber = 0
+            for(var i = 0; i < tokenHoldersPanel.model.count; i ++) {
+                var item = tokenHoldersPanel.model.get(i)
+                if(item.selfDestruct) {
+                    tokensNumber += item.selfDestructAmount
+                }
+            }
+        }
+    }
+
+    title: qsTr("Remotely self-destruct %1 token").arg(root.collectibleName)
+    implicitWidth: 600 // by design
+    topPadding: Style.current.padding
+    bottomPadding: topPadding
+    implicitHeight: Math.min(tokenHoldersPanel.implicitHeight + d.getVerticalPadding() + root.header.height + root.footer.height, d.maxHeight)
```

Comment:
This should be simplified to sth like: `Math.max(root.implicitHeight, d.maxHeight)`

Reply:
mmm.. not sure why.. I really want that the `implicitHeigh` takes into account also the header/footer and paddings, otherwise, always it will become a scrollview popup.

Reply:
It will do this automatically, see here (this is Popup.qml):

```qml
    implicitHeight: Math.max(implicitBackgroundHeight + topInset + bottomInset,
                             contentHeight + topPadding + bottomPadding)
```

What's wrong with the content being scrollable? You're limiting the overall height with `d.maxHeight` anyway